.circleci/config.yml

```diff
@@ -0,0 +1,216 @@
+version: 2
+jobs:
+  "ut-hosted":
+    docker:
+      - image: stronglytyped/xpcc-build-arm:latest
```

Comment:
Do we want to move this docker image to this account? Maybe with a fork? Would like to have this self contained.

Reply:
Probably, but I would like to rename it to something like `xpcc-build` before, getting rid of the `arm` because I recently added the AVR toolchain.

Reply:
BTW, it's like a Matryoshka doll:
```
stronglytyped/xpcc-build-arm <- stronglytyped/arm-none-eabi-gcc <- ubuntu:16.04
```

Reply:
Ok, I _think_ you can fork to this account, ping me if you can’t.

Reply:
I just did. Wondering if there is already a hub.docker.com account associated with RCA? 

Private and strictly-confidential: ;-) Whose email address is used for the github.com/roboterclubaachen account? Should the same address be used for hub.docker.com account?

Reply:
@RoboterClubAachen is an organization and as such must be owned by at least one account. It therefore doesn't have its own email. Nonetheless, it *does* have a billing email, which is our org email (see [our profile info](https://github.com/organizations/roboterclubaachen/settings/profile)). 

You should create an [organiziation on Docker Hub](https://docs.docker.com/docker-hub/orgs/) so that ownership isn't tied to a specific account for life (analog reasoning with this GitHub org).

Reply:
Oh, and you've been upgraded to owner. Happy responsibiliting…

Reply:
Just found the organization on Docker Hub and just created us. ;-)

Reply:
Needed some fiddling to get the accounts linked. Finally, I deleted everything and reconnected it.